src/main/kotlin/app/CacheService.kt

```diff
@@ -68,6 +68,12 @@ object CacheService {
         return null
     }
 
+    fun getUserFromCache(username: String): UserProfile? {
```

Comment:
Added a new function and removed JSON deserialization from [`lookUpInCache`](https://github.com/tipsy/profile-summary-for-github/pull/113/commits/caaa0973bfd645d877f07aabd2d12c4360844d91#diff-50edd97fe3c397a256ba5e035a23f92fbdeac2dffbf60e80baa32021e2b43577R36).

Function [`canLoadUser`](https://github.com/tipsy/profile-summary-for-github/pull/113/files#diff-e05574778ecedcfa59fc66a92874eecfaa6d8be22f7c28bbec89464ae54b3fe6R25-R33) used to deserialize  the JSON only to decide if it's null.